src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -905,7 +904,14 @@ private void ExecuteUninstallPackageCommand(object sender, ExecutedRoutedEventAr
 
             var action = new UserAction(NuGetProjectActionType.Uninstall, package.Id, package.InstalledVersion);
             ExecuteAction(
-                action,
+                () =>
+                {
+                    return Model.Context.UIActionEngine.PerformActionAsync(
+                        Model.UIController,
+                        action,
+                        this,
+                        CancellationToken.None);
+                },
                 nugetUi =>
                 {
                     SetOptions(nugetUi, _detailModel.Options);
```

Comment:
SetOptions() now also sets the projects property. So the 2 lines are all in the new SetOptions() method.